playbooks/ps_81_upgrade.yml

```diff
@@ -0,0 +1,279 @@
+
+  - name: upgrade Percona Server to new deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: no
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-client
+      - percona-server-dbg
+      - percona-server-source
+      - percona-server-rocksdb
+    when: ansible_os_family == "Debian"
+
+  - name: upgrade Percona XtraBackup to new deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: no
+      state: latest
+    vars:
+      packages:
+      - percona-xtrabackup-81
+      - percona-xtrabackup-test-81
+      - percona-xtrabackup-dbg-81
+    when: ansible_os_family == "Debian"
+
+  - name: upgrade Percona release to new rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-release
+    when: ansible_os_family == "RedHat"
+
+  - name: upgrade Percona Server to new rpm packages on CentOS
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-debuginfo
+      - percona-server-devel
+      - percona-server-rocksdb
+      - percona-server-shared
+      - percona-server-shared-compat
+      - percona-server-client
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version != "9"
+
+  - name: upgrade Percona Server to new rpm packages on Oracle Linux 9
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-debuginfo
+      - percona-server-devel
+      - percona-server-rocksdb
+      - percona-server-shared
+      - percona-server-client
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "9"
+
+  - name: upgrade Percona XtraBackup to new rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-xtrabackup-81
+      - percona-xtrabackup-test-81
+      - percona-xtrabackup-81-debuginfo
+    when: ansible_os_family == "RedHat"
+
+  - name: install percona-mysql-shell package
+    include_tasks: ../tasks/install_pshell.yml
+    when: install_mysql_shell == "yes" or install_mysql_shell == ""
+
+  - name: list installed packages
+    include_tasks: ../tasks/list_installed_packages.yml
+
+  - name: check telemetry (enabled)
+    command: /package-testing/check_telemetry.sh ps -e  
+
+  - name: check that Percona Server version is correct
+    command: /package-testing/version_check.sh ps81
+
+  - name: check that Percona Server package versions are correct
+    command: /package-testing/package_check.sh ps81
+
+  - name: check the data masking component
+    command: /package-testing/scripts/ps_dm_component.sh
+
+  - name: check the audit log v2 plugin
+    command: /package-testing/scripts/ps_audit_log_v2_component.sh
+
```

Comment:
Add task to check telemetry:
 - name: check telemetry (disabled)
    command: /package-testing/check_telemetry.sh ps -d